modules/Microsoft.Sql/servers/deploy.bicep

```diff
@@ -94,6 +94,9 @@ var identity = identityType != 'None' ? {
 
 var enableReferencedModulesTelemetry = false
```

Comment:
If possible, please also update the `common` test in the `.test` folder with a test deployment of the encryption protector

Reply:
Note: This will also require to run the `Set-ModuleReadMe` script again after